csm/core/agent/api.py

```diff
@@ -368,8 +377,7 @@ def run(port: int, https_conf: ConfSection, debug_conf: DebugConf):
         else:
             ssl_context = None
 
-        web.run_app(CsmRestApi._app, port=port, ssl_context=ssl_context,
-                    access_log=Log.logger, access_log_format=const.REST_ACCESS_FORMAT)
+        web.run_app(CsmRestApi._app, port=port, ssl_context=ssl_context)
```

Comment:
`access_log` should still be `None`, I think.

Here is a quote from https://docs.aiohttp.org/en/stable/web_reference.html#aiohttp.web.run_app . Notice that `access_log` has `aiohttp.log.access_logger` as its default value, not `None`.

> `aiohttp.web.run_app(app, *, host=None, port=None, path=None, sock=None, shutdown_timeout=60.0, ssl_context=None, print=print, backlog=128, access_log_class=aiohttp.helpers.AccessLogger, access_log_format=aiohttp.helpers.AccessLogger.LOG_FORMAT, access_log=aiohttp.log.access_logger, handle_signals=True, reuse_address=None, reuse_port=None)`
> (...)
> - **`access_log`** – `logging.Logger` instance used for saving access logs. Use `None` for disabling logs for sake of speedup.

Reply:
Good catch, Set access_log to None